specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2022-11-15/mongorbac.json

```diff
@@ -406,7 +406,7 @@
           "description": "A user-friendly name for the Role Definition. Must be unique for the database account."
         },
         "type": {
-          "type": "string",
+          "type": "int",
           "enum": [
```

Comment:
May I ask whether 0 means "BuiltInRole" and 1 means "CustomRole"? If yes, does it make sense to update enum values to 0 and 1?

Reply:
We have to change this to [0, 1] otherwise the generated SDK is still recognizing this as strings

Reply:
Done